cypress_shared/pages/page.ts

```diff
@@ -60,6 +60,54 @@ export default abstract class Page extends Component {
     cy.get(`select[id="${id}"]`).children('option').contains(exact(contents)).should('be.selected')
   }
 
+  shouldShowDateInputsByLegend(legend: string, date: string): void {
```

Comment:
nice

Reply:
I was put off doing this before as I don't have time to go through everything and make it super consistent, but I think I need to just get this in and we start using this approach where we can